oid_validation/test/data/pulp_ssl.cnf

```diff
@@ -0,0 +1,84 @@
+[ req ]
+distinguished_name	= req_distinguished_name
+
+[ ca ]
+default_ca = pulp_ca
+
+[ pulp_ca ]
+x509_extensions	= usr_cert
+
+[ req_distinguished_name ]
+countryName			= Country Name (2 letter code)
+countryName_default		= US
+countryName_min			= 2
+countryName_max			= 2
+
+stateOrProvinceName		= State or Province Name (full name)
+stateOrProvinceName_default	= North Carolina
+
+localityName			= Locality Name (eg, city)
+localityName_default	= Raleigh
+
+0.organizationName		= Organization Name (eg, company)
+0.organizationName_default	= Red Hat
+
+organizationalUnitName		= Organizational Unit Name (eg, section)
+organizationalUnitName_default	= Pulp
+
+commonName			= Common Name (e.g. server FQDN or YOUR name)
+commonName_max			= 64
+commonName_default		= pulp-server
+
+emailAddress			= Email Address (leave this blank)
+emailAddress_max		= 64
+
+[ usr_cert ]
+subjectAltName = @alt_names
+basicConstraints=CA:FALSE
+nsComment			= "OpenSSL Generated Certificate"
+subjectKeyIdentifier=hash
+authorityKeyIdentifier=keyid,issuer
```

Comment:
Need to clean up the whitespace a little, it's obvious where I copied from openssl.cnf vs. wrote it myself